public/js/testimonials.js

```diff
@@ -58,10 +58,18 @@ function loadTestimonials(sortOrder) {
     var items = [];
 
     var gridContent="", index, item;
-    
+    var cur_url = window.location.href;
+    var url_list = cur_url.split("/");
+    var flag = url_list[url_list.length - 1];
+    var load_url = "/documentation/community/testimonials.json";
+    if (flag === "English") {
```

Comment:
well this part looks a little bit wired, is it could be "flag === 'Chinese'" then do the replacement?